src/main/kube-auth-proxy/create-kube-auth-proxy.injectable.ts

```diff
@@ -17,6 +18,7 @@ const createKubeAuthProxyInjectable = getInjectable({
     const proxyPath = isDevelopment ? path.join("client", process.platform, process.arch) : process.arch;
     const dependencies = {
       proxyBinPath: path.join(di.inject(directoryForBundledBinariesInjectable), proxyPath, binaryName),
+      proxyCertPath: di.inject(getKubeAuthProxyCertDirInjectable),
```

Comment:
Name `getKubeAuthProxyCertDirInjectable` implies function, but it is used as a value here.

Reply:
What would be better naming convention for values? `createKubeAuthProxyCertDirInjectable`?